Televi/Controllers/MovieInformationViewController.swift

```diff
@@ -122,8 +136,17 @@ extension MovieInformationViewController {
 }
 
 extension MovieInformationViewController: MovieInformationViewProtocol {
-    func displayMovieInformation(movieInformation: MovieInformation) {
-        self.movieInformation = movieInformation
-        loadingView.isHidden = true
+    func updateScreen(from state: MovieInformationViewState) {
```

Comment:
Change "from state" to "to state", that will make more sense when you're calling the function